Cargo.toml

```diff
@@ -4,10 +4,14 @@ members = [
 	"typify-impl",
 	"typify-macro",
 	"typify-test",
+	"cargo-typify",
 	"example-build",
 	"example-macro",
 ]
 
+[workspace.dependencies]
+schemars = "0.8.11"
+
```

Comment:
This is unrelated but I had just done this for a personal project and got excited 🤣  Happy to rollback if you want!

Saw that `schemars` was in the Cargo.toml for a few of these crates, and thought it might be nice to use the newish workspace dependencies feature to only have to have a version in one place, and make sure all the crates are using the same version

https://doc.rust-lang.org/cargo/reference/workspaces.html#the-dependencies-table

Reply:
Yeah, cool! I would like to handle all dependencies this way. I think it might be useful to do that as a separate PR to avoid merge conflicts.

Reply:
Reverted for now